pkg/controller/migmigration/quiesce.go

```diff
@@ -131,6 +134,12 @@ func (t *Task) quiesceDeploymentConfigs(client k8sclient.Client) error {
 				continue
 			}
 			dc.Annotations[ReplicasAnnotation] = strconv.FormatInt(int64(dc.Spec.Replicas), 10)
+			t.Log.Info(fmt.Sprintf("Quiescing DeploymentConfig [%v/%v]. "+
```

Comment:
I think we may want to file an issue for this, but a call to the scale subresource may be better than updating each individual GVK and will capture CRD's that have the scale subresource mapped to a value so we can scale down those applications as well. 

We could use discovery API to determine if a given GVK has a scale subresource.

Reply:
Considering we also need to issue an UPDATE to add an annotation with old replica count at the same time, is there still improvement to be had from changing this to using scale? We currently do:

1. Replica count change
2. Annotation add

In a single reconcile.

Reply:
Why do we have to set that annotation?  sorry if I am missing it.

Reply:
We need to keep track of which resources we've quiesced and their previous replica counts so that we can:
1. Come back later and unquiesce to previous replica count them as part of usual migration routine, since the resources land on destination in quiesced
2. Unquiesce source cluster to appropriate replica count on rollback